extensions/objects/templates/skill-selector-editor.component.ts

```diff
@@ -98,6 +100,14 @@ export class SkillSelectorEditorComponent implements OnInit, OnDestroy {
 
   ngOnDestroy(): void {
     this.contextService.removeCustomEntityContext();
+    /**
+     * Note to developers:
+     * We are manually setting customEntityContext so that it
+     * doesn't cause any breakages in question-editor.
+     * See issue #16985 for detailed discussion.
+     */
+    this.contextService.setCustomEntityContext(
+      AppConstants.IMAGE_CONTEXT.QUESTION_SUGGESTIONS, this.skillId);
```

Comment:
I wonder, instead of defaulting to this context, would it make sense to store the customContext in some manner before setting it in ngOnInit and then restore that context on ngOnDestroy?

Reply:
Yup that is the same thing but in a better way ig. Done PTAL.